backend/lib/endpoints/posts.js

```diff
@@ -52,18 +52,30 @@ async function routes(app) {
           limit,
           objective,
           skip,
+          postMode,
           includeMeta,
         },
       } = req;
+      
       const queryFilters = filter ? JSON.parse(decodeURIComponent(filter)) : {};
       // Base filters - expiration and visibility
       /* eslint-disable sort-keys */
-      const filters = [
+      let filters = [
         {
           $or: [{ expireAt: null }, { expireAt: { $gt: new Date() } }],
           status: { $ne: "removed" },
         },
       ];
+      // "IA" stands for InActive or Archived
+      //In future we will also have "D" for Drafts
+      if(postMode === 'IA'){
```

Comment:
this would be a good use case of enum type, and it's better to let the code explain itself (e.g. define an enum of POST_MODE = ['inactive', 'draft'] inside the `lib/endpoints/schema/posts.js`, so the input can also be validated)